src/lib/types/CommonInputProps.ts

```diff
@@ -12,6 +12,8 @@ interface CommonInputProps<T extends FieldValues> {
   labelToolTip?: string;
   markAllOnFocus?: boolean;
   inputOnly?: boolean;
+  className?: React.HTMLAttributes<T>["className"];
```

Comment:
I'm aware that it hasn't been done so far but could you please document those props here? 

Please use jsdoc. Something like this:

https://github.com/neolution-ch/react-hook-form-components/blob/main/src/lib/types/CommonInputProps.ts#L26

Please also mention that for the both properties it's additional classNames or style. (Just so the developer is aware that it won't override the existing classes)